### PR TITLE
[14.0] [mig] hr_utilization_analysis

### PR DESCRIPTION
**Pull request by schhatbar-initos**

### CODE REVIEW

**Review by dsolanki-initos — APPROVED** [review]
LGTM

**Comment by schhatbar-initos** [comment]
@davejames Can you please review this PR?
Thanks in Advance!

**Comment by schhatbar-initos** [comment]
@alexey-pelykh Can you please review this PR?

**Comment by schhatbar-initos** [comment]
@all can you please review the PR ?

**Comment by dsolanki-initos** [comment]
@all can somone please review the PR ?

**Comment by alexey-pelykh** [comment]
@dsolanki-initos you do realize that `@ALL` has no relation to OCA?

**Comment by dsolanki-initos** [comment]
> @dsolanki-initos you do realize that `@ALL` has no relation to OCA?

Yes I know I don't want to tag all.It is for all who looks at the PR and if someone can review it.

**Comment by dsolanki-initos** [comment]
It would be great somone cna add reviews here and merge the PR.

**Comment by dsolanki-initos** [comment]
It would be nice if we could get reviews here.

**Comment by dsolanki-initos** [comment]
It would be nice if we could get reviews here.

File: hr_utilization_analysis/report/hr_utilization_analysis.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                    ("employee_id", "=", entry.employee_id.id),
+                    ("date", "=", entry.date),
+                ]
+            )
+
+    @api.depends("employee_id", "date")
+    def _compute_capacity(self):
+        Module = self.env["ir.module.module"]
+
+        project_timesheet_holidays = Module.with_user(SUPERUSER_ID).search(
```
Comment (MeetKD) [inline hr_utilization_analysis/report/hr_utilization_analysis.py]: SUPERUSER_ID -> sudo

**Review by MeetKD — CHANGES_REQUESTED** [review]
Non blocking minor change. I have rebuild the runbot build to check.

**Comment by dsolanki-initos** [comment]
It would be great if the maintainers can merge this PR.

**Comment by dsolanki-initos** [comment]
@OCA/human-resources-maintainers It would be great if we could merge this PR

**Review by Saran440 — APPROVED** [review]
Code Reviewed, No Change :+1:

**Comment by dsolanki-initos** [comment]
@MeetKD  you can review it again I have udpated the changes suggested.

File: hr_utilization_analysis/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_my_wizard_user,my.wizard.user,model_hr_utilization_analysis_wizard,hr.group_hr_manager,1,1,1,1
```
Comment (alexey-pelykh) [inline hr_utilization_analysis/security/ir.model.access.csv]: I'm not sure group is correct. Menu item is located under `hr_timesheet.menu_timesheets_reports`, and in v14 it has `hr_timesheet.group_timesheet_manager` group

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: hr_utilization_analysis/security/ir.model.access.csv (hunk 2 above)
Comment (dsolanki-initos) [reply hr_utilization_analysis/security/ir.model.access.csv]: Yes to view the menu user must have Timesheet manager as well as HR manger rights.

Comment (alexey-pelykh) [reply hr_utilization_analysis/security/ir.model.access.csv]: I don't follow you. Parent menu is accessible to `hr_timesheet.group_timesheet_manager`, why then this wizard requires `hr.group_hr_manager`?

**Comment by dsolanki-initos** [comment]
It would be great if maintainers could merge it

**Comment by dreispt** [comment]
@dsolanki-initos I believe a requested change is yet to be addressed.

**Comment by dsolanki-initos** [comment]
Hi @dreispt 
I've implemented the requested change can you please check it?

**Comment by dsolanki-initos** [comment]
It would be great if maintainers could merge it

**Comment by hkapatel-initos** [comment]
Hello, @dreispt  It would be nice if you are trying to merge it.

**Pull request merged**